PepperDashEssentials/Room/Types/EssentialsHuddleVtc1Room.cs

```diff
@@ -672,6 +692,35 @@ public static void AllRoomsOff()
 				(room as EssentialsHuddleSpaceRoom).RunRouteAction("roomOff");
 		}
 
+
+		/// <summary>
+		/// Setup the external sources for the Cisco Touch 10 devices that support IHasExternalSourceSwitch
+		/// </summary>
+		private void SetCodecExternalSources()
+		{
+			
+			string codecTieLine = "";
+			codecTieLine = ConfigReader.ConfigObject.TieLines.SingleOrDefault(x => x.DestinationKey == VideoCodec.Key).DestinationPort;
+			(VideoCodec as IHasExternalSourceSwitching).ClearExternalSources();
+			(VideoCodec as IHasExternalSourceSwitching).RunRouteAction = RunRouteAction;
+			var srcList =  ConfigReader.ConfigObject.SourceLists.SingleOrDefault(x => x.Key == SourceListKey).Value.OrderBy(kv => kv.Value.Order);;
+
+				foreach (var kvp in srcList)
+				{
+					var srcConfig = kvp.Value;
+					
+					if (kvp.Key != "codecOsd" && kvp.Key != "roomOff")

```

Comment:
make the variables `codecOsd` and `roomOff` be constants for this class. This way, if they ever change in the future, we only have to change them in one spot.

Reply:
I see `public string DefaultCodecRouteString { get { return "codecOsd"; } } ` which i have adopted in the conditional check. The term "roomOff" is not a constant in the class EssentialsHuddleVtc1Room.cs, just a reused string thoughout the class in various places. Changing it is beyond the scope of this effort.